doc/dependencies.rst

```diff
@@ -12,7 +12,7 @@ Security aspects
 ----------------
 
 A project that uses Hunter first downloads the hunter package itself. This is
-done with a help of HunterGate CMake module. A copy of the HunterGate module
+done with the help of HunterGate CMake module. A copy of the HunterGate module
 is included in the main project. The HunterGate is ordered to download specified
```

Comment:
Can you add "module" after "HunterGate" and "the" between "download" and "specified" i.e.:

"The HunterGate module is ordered to download the specified"